cdk/lib/report.ts

```diff
@@ -17,5 +35,79 @@ export class Report extends GuStack {
 		new CfnInclude(this, 'YamlTemplate', {
 			templateFile: yamlTemplateFilePath,
 		});
+		const app = 'report';
+		const playApp = new GuPlayApp(this, {
+			access: {
+				scope: AccessScope.PUBLIC,
+			},
+			app,
+			certificateProps: {
+				domainName: props.domainName,
+			},
+			instanceMetricGranularity: props.instanceMetricGranularity,
+			instanceType: InstanceType.of(InstanceClass.T4G, InstanceSize.MICRO),
+			// This matches the YAML stack (i.e. there is no 5XX alarm).
+			// There is no slo-alert config for this service either (presumably due to the traffic level), so we have an
+			// observability gap here.
+			monitoringConfiguration: { noMonitoring: true },
```

Comment:
We ought to do something about this but we're not making anything worse (when comparing to the legacy infrastructure), so I don't plan to address it in this PR.